unittests/testmain.c

```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+
+void *kmalloc(size_t);
+
+int main(int argc, char **argv) {
+
+   printf("hello from the kernel unit tests!\n");
+   printf("kmalloc(10) returns: %p\n", kmalloc(10));
+
```

Comment:
That was not a test. That's why I did not use any assert.
My point was to try calling an independent kernel function from the tests and see that it works.